### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by joeratt**
http://fhir.cerner.com is currently referencing some missing links on https://code.cerner.com. Proposing an update to point to the EHR API page.

### CODE REVIEW

**Comment by daliboz** [comment]
Hey Joe - we were working on this/it's in our queue. If you want, we can update these changes to be what we have currently proposed. I'll make the comments.

File: layouts/header.html [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
       <nav id="navigation">
         <ul>
```
Comment (daliboz) [inline layouts/header.html]: We're going to remove all these links in the Header. Update the code image to point to the homepage of code.cerner.com instead.

File: content/index.md [hunk 2]
```diff
@@ -15,6 +15,6 @@ layout: overview
 <div class="full-width dev-program-callout">
   <div class="wrapper">
     <h2>Build a business relationship with Cerner around our implementation of the SMART and FHIR<sup>®</sup> standards.</h2>
-    <p>Interested in collaborating? <a href="https://code.cerner.com/build">Learn more</a>.</p>
```
Comment (daliboz) [inline content/index.md]: Update to: https://code.cerner.com/ehr-api#howitworks

File: content/millennium/faq.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ title: FAQs
 
 ### What is Cerner's roadmap for FHIR<sup>®</sup> standard implementation?
 
-Both the deprecated May2015 and current DSTU 2 Final versions of our HL7 FHIR<sup>®</sup> standard implementation are available in production. Our roadmap always includes helping provide feedback to the current draft of the FHIR<sup>®</sup> standard, as well as updating to the latest release. Additionally, our roadmap is prioritized depending on maturity of the resources in the FHIR<sup>®</sup> standard (in other words, how stable they are considered), as well as the needs of our business partners. Some external forces, like Meaningful Use, also help determine the priorities and order for the FHIR<sup>®</sup> standard resources we implement. If you would like to create a business partnership with us and help guide our roadmap, please read the [How It Works Section](https://code.cerner.com/build) on code.cerner.com. Once an application has been tested in our secure sandbox, fill out [this form](https://cernermarketing.wufoo.com/forms/z352gae0xcax6t/).
```
Comment (daliboz) [inline content/millennium/faq.md]: Update to https://code.cerner.com/ehr-api#howitworks

**Comment by joeratt** [comment]
@daliboz - I made the changes you've suggested. I can close the PR if you'd prefer to tackle this with the item in your queue.

File: layouts/header.html (hunk 1 above)
Comment (joeratt) [inline layouts/header.html]: I wasn't sure if you were proposing to remove the nav itself - I've updated the Image href so far.

Comment (daliboz) [inline layouts/header.html]: Yes, completely remove the nav section.

**Comment by daliboz** [comment]
No, we'll take the help ;) 

Thanks, btw

**Comment by MaxPhilips** [comment]
+1 after nav removed

**Comment by daliboz** [comment]
+1, you can merge. Thanks @joeratt!!

**Pull request merged**